command/crypto/jwk/thumbprint.go

```diff
@@ -28,7 +28,7 @@ For examples, see **step help crypto jwk**.`,
 	}
 }
 
-func thumbprintAction(ctx *cli.Context) error {
+func thumbprintAction(_ *cli.Context) error {
```

Comment:
We can get rid of it or:
```suggestion
func thumbprintAction(*cli.Context) error {
```